rr_gazebo/scripts/spawn_macaroni_platform.sh

```diff
@@ -3,3 +3,7 @@
 sleep 3
 
 roslaunch rr_gazebo spawn_macaroni_platform.launch x:=$1 y:=$2 heading:=$3
+
+# set gazebo physics step size so that it uses less CPU resources
+sleep 2
+gz physics --step-size 0.0025 --update-rate 400 --iters 30
```

Comment:
You can do this directly in the launch file.

Reply:
I tried that but then the settings have to be set in all 4 world models that we have. That's not such a bad thing but I'm not sure which is better